en/developer/reference/articles/02-sample-projects/02-apps/service-builder-samples/service-builder-jndi.markdown

```diff
@@ -6,18 +6,45 @@ header-id: service-builder-application-using-external-database-via-jndi
 
 [TOC levels=1-4]
 
-This sample demonstrates how to connect a Liferay Service Builder application to
-an external database via a JNDI connection. Here, an external database means any
+The `apps/service-builder/jndi` sample demonstrates how to connect a Liferay Service Builder application to
+an external database via a JNDI connection configured on the application server. Here, an external database means any
 database other than @product@'s database. For this sample to work correctly, you
-must prepare such an external database and configure @product@ to use it. Follow
-the steps below to make the required preparations before deploying the
+must prepare such an external database and configure your application server to use it.
+
+| **Important:** Connecting to an external data source using JNDI is broken in
+| Portal CE 7.2 GA1 and GA2, and in DXP 7.2 releases prior to FP5/SP2.
+
+Follow the steps below to make the required preparations before deploying the
 application.
```

Comment:
Improved steps to really focus on what the user must add in their environment to work with the sample.